src/test/java/com/endava/webapp/controller/EmployeeControllerTest.java

```diff
@@ -0,0 +1,95 @@
+package com.endava.webapp.controller;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.http.MediaType;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+class EmployeeControllerTest extends BaseIT {
+
+    @Test
+    void shouldReturnOk_whenGettingAllEmployees() throws Exception {
+        mockMvc.perform(get("/employees?offset=2&limit=3"))
+                .andExpect(status().isOk());
+    }
+
+    @Test
+    void shouldReturnOk_whenGettingExistingEmployeeById() throws Exception {
+        mockMvc.perform(get("/employees/1"))
+                .andExpect(status().isOk());
```

Comment:
Not need to fix it right now, but in the project please assert the actual response with expected.

Reply:
Okay, thanks